roles/nf-core/defaults/main.yml

```diff
@@ -0,0 +1,21 @@
+nextflow_java: "/sw/comp/java/x86_64/sun_jdk1.8.0_151"
+nextflow_version_tag: "v19.04.1"
+nextflow_download_url: "https://github.com/nextflow-io/nextflow/releases/download/{{ nextflow_version_tag }}/nextflow"
+nf_core_env: "/lupus/ngi/irma3/nf-core-env"
+pipelines:
+  - name: rnaseq
+    release: 1.4.2
+    containers:
+      - nf-core-rnaseq-1.4.2.simg
```

Comment:
Can we use `{{ release }}` in the container name? Could avoid accidental mismatches in the future..

Reply:
Or indeed we could presumably autogenerate this filename at a later date wherever it is used:
```
nf-core-{{ pipeline_name }}-{{ pipeline_version }}.simg
```

Reply:
How is this thought to work for pipelines with multiple containers? Am I right in that nf-core download currently doesn't work for those? Is there a plan for that?

Reply:
Yes, you are correct. The plan is to build support for this in to `nf-core download`, it should be possible (and hopefully not too difficult). It's a fairly low priority for now though as it's not needed for many pipelines as it stands.

My suggestion was to keep the `sarek` role separate for now as it needs custom behaviour here. In the long run I would like to add this support in nf-core download and keep as much logic in there as possible to avoid having to hardcode anything here.

Reply:
> Can we use `{{ release }}` in the container name? Could avoid accidental mismatches in the future..

Bump 😄

Reply:
Removed the container list!